x-pack/plugins/maps/public/classes/sources/es_source/es_source.ts

```diff
@@ -448,7 +448,7 @@ export class AbstractESSource extends AbstractVectorSource implements IESSource
       searchSource.setField('query', sourceQuery);
     }
     if (style.isTimeAware() && (await this.isTimeAware())) {
-      const timeFilter = getTimeFilter().createFilter(indexPattern, timeFilters);
+      const timeFilter = getTimeFilter().createFilter(indexPattern, { timeRange: timeFilters });
```

Comment:
I'm not a big fan of changing the function signature when there are a few places that don't require the new paramaeter. TimeRange` already has a `mode` parameter. Is there a reason why this new `coreceConversion` parameter wasn't added on the `TimeRange` itself?

Reply:
@jloleysens how about adding a new method to the Timefilter service, such as `Timefilter.createUnconvertedFilter`, which can be used to fix the issue in Discover app? I agree with Maja that a lot of places use this existing method, so it's probably best to not change it, at least for this PR.

Reply:
@tsullivan I like that idea 👍🏻